build/scripts/tests/run_testworld.sh

```diff
@@ -8,7 +8,7 @@ eval "$cleanup"
 
 status=$?
 
-output="go test -race -coverprofile=profile.out -covermode=atomic -tags=testworld github.com/centrifuge/go-centrifuge/testworld 2>&1"
+output="go test -coverprofile=profile.out -covermode=atomic -tags=testworld github.com/centrifuge/go-centrifuge/testworld 2>&1"
```

Comment:
not running with race conditions anymore ?

Reply:
I would love to have this, just that it requires around 3gb memory and travis just keeps timing out randomly because of that.

Reply:
reverted, it worked this time.

Reply:
Nice!